validation/validation_test.go

```diff
@@ -0,0 +1,67 @@
+package validation_test
+
+import (
+	"testing"
+
+	"github.com/preslavmihaylov/todocheck/validation"
+	"github.com/preslavmihaylov/todocheck/config"
+)
+
+func TestInvalidOrigins(t *testing.T) {
+	invalidConfigPaths := []string{
+		"../testing/test_configs/invalid_github_https.yaml",
+		"../testing/test_configs/invalid_github_origin.yaml",
+		"../testing/test_configs/invalid_github_www.yaml",
+		"../testing/test_configs/invalid_gitlab_origin.yaml",
+		"../testing/test_configs/invalid_gitlab_port.yaml",
+		"../testing/test_configs/invalid_issue_tracker.yaml",
+		"../testing/test_configs/invalid_jira_origin.yaml",
+		"../testing/test_configs/invalid_jira_port.yaml",
+		"../testing/test_configs/invalid_offline_url.yaml",
+		"../testing/test_configs/invalid_pivotal_origin.yaml",
+		"../testing/test_configs/invalid_redmine_origin.yaml",
+		"../testing/test_configs/invalid_redmine_port.yaml",
+	}
+
+	for _, path := range invalidConfigPaths {
+		cfg, err := config.NewLocal(path, ".")
+		if err != nil {
+			t.Errorf("%s", err)
+			continue
+		}
```

Comment:
Using `config.NewLocal` is not a good idea given that we're writing a unit test in the `validation` package. Here, the tests should only test the `validation` package, instead of also testing the rest.

Instead, what you can do is:
```
cfg := config.Local{
  Origin: <your-test-origin>,
  IssueTracker: <your-test-issue-tracker>
}

errors := validation.Validate(cfg)
...
```

This way, you are testing the `validation package in isolation and what's more, you avoid having to create all those extra files.

So, to sum this up, we'll have to create a test case for each of those files (not necessarily in separate functions) and delete those files if they're no longer used in the `testing` directory.